core/src/main/res/layout/main_activity_content.xml

```diff
@@ -20,14 +20,15 @@
 
       <include layout="@layout/layout_toolbar" />
 
-      <org.kiwix.kiwixmobile.core.main.AnimatedProgressBar
+      <ProgressBar
         android:id="@+id/activity_main_progress_view"
         android:layout_width="match_parent"
         android:layout_height="@dimen/progress_view_height"
+        android:max="100"
+        android:progress="1"
```

Comment:
I don't think it needs a starting value of 1, probably best to omit it

Reply:
Done👍